lib/storage/src/content_manager/toc/create_collection.rs

```diff
@@ -156,11 +156,16 @@ impl TableOfContent {
             Some(diff) => diff.update(&self.storage_config.wal)?,
         };
 
-        let optimizers_config = match optimizers_config_diff {
+        let mut optimizers_config = match optimizers_config_diff {
             None => self.storage_config.optimizers.clone(),
             Some(diff) => diff.update(&self.storage_config.optimizers)?,
         };
 
+        // Apply global optimizers-config overwriting if configured
+        if let Some(optimizers_overwrite) = &self.storage_config.optimizers_overwrite {
+            optimizers_config = optimizers_overwrite.clone().update(&optimizers_config)?;
+        }
```

Comment:
This change will create collection with "optimizers_overwrite" parameters, which will be just persisted everywhere.

But what we need is different: we need to use `optimizers_overwrite` _even though_ the collection is configured with different values.
That means, for example, that we should be able to restart service with differnt config, and the already existing collection should act differently, without changes of the persisted collection config